cmd/pbm/backup.go

```diff
@@ -191,11 +192,20 @@ func runBackup(
 	}
 
 	if b.wait {
+		if b.waitTime > time.Second {
```

Comment:
I'd allow any positive value e.g. 500ms. Currently that value will be just ignored, but in opposite case user will get deadline exceeded error which is more clear.